torchtitan/experiments/deepseek_v3/model_config.py

```diff
@@ -132,11 +132,19 @@ class ModelArgs:
     aux_loss_alpha: float = 0.001
     seq_aux: bool = True
     hidden_act: str = "silu"
-    max_position_embeddings: int = 4096
+    max_position_embeddings: int = 163840
     initializer_range: float = 0.02
     rms_norm_eps: float = 1e-6
     rope_theta: float = 10000.0
-    rope_scaling = None
+    rope_scaling: dict = field(default_factory=lambda: {
+        "beta_fast": 32,
+        "beta_slow": 1,
+        "factor": 40,
+        "mscale": 1.0,
+        "mscale_all_dim": 1.0,
+        "original_max_position_embeddings": 4096,
+        "type": "yarn"
+    })
```

Comment:
Thank you so much for filling in these settings! 
I wonder if you'd be able to write a short comment on what these settings refer to or where they come from? Thank you!